bitnami/airflow/templates/_helpers.tpl

```diff
@@ -348,20 +348,37 @@ Compile all warnings into a single message, and call fail.
 {{- end -}}
 {{- end -}}
 
-{{/* Validate values of Airflow - "airflow.cloneDagFilesFromGit.repository" must be provided when "airflow.cloneDagFilesFromGit.enabled" is "true" */}}
-{{- define "airflow.validateValues.cloneDagFilesFromGit.repository" -}}
-{{- if and .Values.airflow.cloneDagFilesFromGit.enabled (empty .Values.airflow.cloneDagFilesFromGit.repository) -}}
-airflow: airflow.cloneDagFilesFromGit.repository
+{{/* Validate values of Airflow - Atleast one repository details must be provided when "airflow.cloneDagFilesFromGit.enabled" is "true" */}}
+{{- define "airflow.validateValues.cloneDagFilesFromGit.repositories" -}}
+{{- if and .Values.airflow.cloneDagFilesFromGit.enabled (empty .Values.airflow.cloneDagFilesFromGit.repositories) -}}
+{{- if or (empty .Values.airflow.clonePluginsFromGit.repository) (empty .Values.airflow.clonePluginsFromGit.branch) -}}
+airflow: airflow.cloneDagFilesFromGit.repositories
+    At least one repository must be provided when enabling downloading DAG files
+    from git repository (--set airflow.cloneDagFilesFromGit.repositories[0].repository="xxx"
+    --set airflow.cloneDagFilesFromGit.repositories[0].name="xxx"
+    --set airflow.cloneDagFilesFromGit.repositories[0].branch="name")
```

Comment:
@dani8art Don't you think we should modify the message here in situation i.e if someone mentioned `airflow.clonePluginsFromGit.repository` and didn't mention `airflow.clonePluginsFromGit.branch` we should let them know that both of them is mandatory not the above `repositories` message?

Reply:
I think this is enough, I mean, we want to avoid someone using last values to upgrade the chart doesn't make errors. However, if someone new wants to use `.Values.airflow.clonePluginsFromGit.repository` the message will redirect him to the new approach.

BTW, this message will also redirect an old user to the new approach